doc/devel/dependencies.rst

```diff
@@ -0,0 +1,156 @@
+.. _dependencies:
+
+============
+Dependencies
+============
+
+Mandatory dependencies
+======================
+
+When installing through a package manager like ``pip`` or ``conda`` the
+mandatory dependencies are automatically installed. This list is mainly for
+reference.
+
+* `Python <https://www.python.org/downloads/>`_ (>= 3.7)
+* `NumPy <https://numpy.org>`_ (>= 1.16)
+* `setuptools <https://setuptools.readthedocs.io/en/latest/>`_
+* `cycler <https://matplotlib.org/cycler/>`_ (>= 0.10.0)
+* `dateutil <https://pypi.org/project/python-dateutil>`_ (>= 2.7)
+* `kiwisolver <https://github.com/nucleic/kiwi>`_ (>= 1.0.1)
+* `Pillow <https://pillow.readthedocs.io/en/latest/>`_ (>= 6.2)
+* `pyparsing <https://pypi.org/project/pyparsing/>`_ (>=2.2.1)
+
+
+.. _optional_dependencies:
+
+Optional dependencies
+=====================
+
+The following packages and tools are not required but extend the capabilities
+of Matplotlib.
+
+Matplotlib figures can be rendered to various user interfaces. See
+:ref:`what-is-a-backend` for more details on the optional Matplotlib backends
+and the capabilities they provide.
+
+* Tk_ (>= 8.3, != 8.6.0 or 8.6.1) [#]_: for the Tk-based backends.
+* PyQt4_ (>= 4.6) or PySide_ (>= 1.0.3) [#]_: for the Qt4-based backends.
+* PyQt5_ or PySide2_: for the Qt5-based backends.
+* PyGObject_: for the GTK3-based backends [#]_.
+* wxPython_ (>= 4) [#]_: for the wx-based backends.
+* pycairo_ (>= 1.11.0) or cairocffi_ (>= 0.8): for the GTK3 and/or cairo-based
+  backends.
+* Tornado_: for the WebAgg backend.
+
+.. _Tk: https://docs.python.org/3/library/tk.html
+.. _PyQt4: https://pypi.org/project/PyQt4
+.. _PySide: https://pypi.org/project/PySide
+.. _PyQt5: https://pypi.org/project/PyQt5
+.. _PySide2: https://pypi.org/project/PySide2
+.. _PyGObject: https://pygobject.readthedocs.io/en/latest/
+.. _wxPython: https://www.wxpython.org/
+.. _pycairo: https://pycairo.readthedocs.io/en/latest/
+.. _cairocffi: https://cairocffi.readthedocs.io/en/latest/
+.. _Tornado: https://pypi.org/project/tornado
+
+.. [#] Tk is part of most standard Python installations, but it's not part of
+       Python itself and thus may not be present in rare cases.
+.. [#] PySide cannot be pip-installed on Linux (but can be conda-installed).
+.. [#] If using pip (and not conda), PyGObject must be built from source; see
+       https://pygobject.readthedocs.io/en/latest/devguide/dev_environ.html.
+.. [#] If using pip (and not conda) on Linux, wxPython wheels must be manually
+       downloaded from https://wxpython.org/pages/downloads/.
+
+For better support of animation output format and image file formats, LaTeX,
+etc., you can install the following:
```

Comment:
The etc is confusing here-either explicit list all the things  since you list almost all of  'em or drop the list since it's below.